client/src/components/common/ContentPage.tsx

```diff
@@ -23,11 +23,6 @@ export default function ContentPage({ title, children }) {
   );
 }
 
-ContentPage.propTypes = {
-  title: PropTypes.string,
-  children: PropTypes.node.isRequired,
-};
-
 ContentPage.defaultProps = {
   title: '',
 };
```

Comment:
I think having `title = ''` in line 17 is the more standard way of doing this, though I'm not sure if it's exactly the same.

Reply:
I tried that and it failed to compile. I think it has to be like `title : '' `.